api/v1alpha1/eventing_types.go

```diff
@@ -24,47 +27,51 @@ import (
 
 // NOTE: json tags are required.  Any new fields you add must have json tags for the fields to be serialized.
 
+// Eventing is the Schema for the eventing API.
+// +kubebuilder:object:root=true
+// +kubebuilder:subresource:status
+// +kubebuilder:printcolumn:name="State",type="string",JSONPath=".status.state",description="State of Eventing"
+// +kubebuilder:printcolumn:name="Age",type="date",JSONPath=".metadata.creationTimestamp",description="Age of the resource"
+type Eventing struct {
+	metav1.TypeMeta   `json:",inline"`
+	metav1.ObjectMeta `json:"metadata,omitempty"`
+
+	// +kubebuilder:default:={backends:{type:"NATS", config:{natsStreamStorageType:"File", natsStreamReplicas:3, natsStreamMaxSize:"700Mi", natsMaxMsgsPerTopic:1000000}}, logging:{logLevel:Info}, publisher:{replicas:{min:2,max:2}, resources:{limits:{cpu:"500m",memory:"512Mi"}, requests:{cpu:"10m",memory:"256Mi"}}}}
+	Spec   EventingSpec   `json:"spec,omitempty"`
+	Status EventingStatus `json:"status,omitempty"`
+}
+
+// EventingStatus defines the observed state of Eventing
+type EventingStatus struct {
+	State      string             `json:"state"`
+	Conditions []metav1.Condition `json:"conditions,omitempty"`
+}
+
 // EventingSpec defines the desired state of Eventing
 type EventingSpec struct {
 	// Backends defines the list of eventing backends to provision.
-	Backends []Backend `json:"backends"`
+	// +kubebuilder:default:={type:"NATS", config:{natsStreamStorageType:"File", natsStreamReplicas:3, natsStreamMaxSize:"700Mi", natsMaxMsgsPerTopic:1000000}}
```

Comment:
Do we need to mention the default values here again? Because it's already mentioned [here](https://github.com/kyma-project/eventing-manager/blob/893018f588092576b31743f1215f8d079f9a8091/api/v1alpha1/eventing_types.go#L39).
Can you please check if we really need it in two places?